Day3/application.py

```diff
@@ -15,9 +15,9 @@ def default():
 
 @app.route('/success/<name>?<pwd>')
 def success(name, pwd):
-   print("name ", name)
-   print("pwd", pwd)
-   return 'welcome ' + name + " "+ pwd
+   # print("name ", name)
+   # print("pwd", pwd)
```

Comment:
remove commented code

Reply:
removed the code sir, will commit after all the changes.